src/towerside/actuators.hpp

```diff
@@ -90,6 +90,112 @@ class Ignition: public I2C {
   Ignition(uint8_t slave_address): I2C(slave_address) {}
 };
 
+class Heater {
+  uint8_t slave_address; // slave address we are controlling
+  bool power_set;
+  uint8_t reg_set;
+
+public:
+
+  Heater(uint8_t slave_address):
+	  slave_address{slave_address},power_set{0},reg_set{0} {}
+
+  void set(bool value) {
+	power_set = value;
+    Wire.beginTransmission(slave_address);
+    // Relay boards have two relays. One turns on power, and the other selects which direction to apply power to.
+    // LSB is power, next bit is select.
```

Comment:
These comments seem wrong, fix pls. Why are we writing the last used reg_set in this function anyways, that seems like its only relevant when reading?

Reply:
Yeah agree since we set register select each time we read anyways